readthedocs/invitations/views.py

```diff
@@ -27,14 +27,14 @@ class RevokeInvitation(PrivateViewMixin, UserPassesTestMixin, DeleteView):
     pk_url_kwarg = "invitation_pk"
     http_method_names = ["post"]
 
-    def delete(self, request, *args, **kwargs):
+    def form_valid(self, form):
```

Comment:
This seems like a functionality change that I don't understand?

Reply:
This was on the release notes: https://docs.djangoproject.com/en/4.2/releases/4.0/#generic-views

Reply:
Confusing, but ok 👍